server/proxy/volumes.go

```diff
@@ -0,0 +1,91 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package proxy
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/docker/compose-cli/aci"
+	"github.com/docker/compose-cli/api/volumes"
+	"github.com/docker/compose-cli/context/store"
+	"github.com/docker/compose-cli/errdefs"
+	volumesv1 "github.com/docker/compose-cli/protos/volumes/v1"
+)
+
+// VolumesCreate creates a volume.
+func (p *proxy) VolumesCreate(ctx context.Context, req *volumesv1.VolumesCreateRequest) (*volumesv1.VolumesCreateResponse, error) {
+	backendClient := Client(ctx)
+	contextType := ""
+	if backendClient != nil {
+		contextType = backendClient.ContextType()
+	}
+
+	if contextType != store.AciContextType {
```

Comment:
Not sure we really want this here, the backend should return a `not implemented` error instead and then the cli can say that the volumes is not implemented. no ?

Reply:
indeed this is not reached if not ACI